assets/styles/04-components/wordlimit.scss

```
@@ -1,8 +1,8 @@
-section {
+#wordlimit {
   display: grid;
   width: 50%;
 }
 
-label {
+#wordlimit > label {
   margin-bottom: 1em;
```

```suggestion
  margin-bottom: var(--theme-spacing--gutter);
```

Please don't use literals - use the existing design tokens.